### ISSUE TITLE
Personal Inventory

### ISSUE THREAD

**Issue by KerstinMaur** [WFCD/navis#94]
**Is your feature request related to a problem? Please describe.**
I am not sure which stuff I already have and what I need. 😅

**Describe the solution you'd like**
There might be some sort of inventory where users can see all warframes, weapons, sentinels and mods and mark the ones they own. ✔️

**Describe alternatives you've considered**
At the moment I am using a spreadsheet (e.g. [LibreOffice Calc](https://www.libreoffice.org/), [Microsoft Excel](https://www.microsoft.com/microsoft-365/) or [Google Spreadsheets](https://spreadsheets.google.com/)). 📋

**Additional context**
I don't have additional context at the moment. 🗅

**Comment by SlayerOrnstein** [comment]
I've always found this to be out of the general scope of app, but if you really need one I've heard good things from this one https://warframe-foundry.app/

For now I'm gonna just be doing so QoL and fixes, when I feel like it's a good spot stability wise then I'll start adding more features, with this one most likely being the first thing added considering how many times I've been asked in the past. No promises.

**Comment by SlayerOrnstein** [comment]
@KerstinMaur @TobiTenno How should we go about this then?

**Comment by TobiTenno** [comment]
we could write our own api for inventory and save it ourselves from that, but then authentication, etc....
could get gross

**Comment by SlayerOrnstein** [comment]
Oof we could always use sqlite or hive, run the DB from local storage.

**Comment by TobiTenno** [comment]
true, just wouldn't persist across sessions, like if they have a phone, tablet, etc. or if we end up in a browser or desktop app someday

**Comment by SlayerOrnstein** [comment]
Would setting something up like appwrite or superbase help then?

**Comment by TobiTenno** [comment]
something like that would work

**Comment by SlayerOrnstein** [comment]
What works better for you? I've never messed with database enough to know how to make something that would scale that well, so having someone with xp to help out would be best on my end.

**Comment by SlayerOrnstein** [comment]
@KerstinMaur hey if you're in the server you can see and chime in on some of the progress so far

**Comment by KerstinMaur** [comment]
Sorry, I was not active for some time, @SlayerOrnstein. 
I love to see the progress. 
Thank you so much. You are aweseome!

**Comment by SlayerOrnstein** [comment]
https://github.com/WFCD/navis/releases/tag/v2.23.0-beta.1

also on live Google Play

@KerstinMaur Not complete because not sure where to go from here or what the user flow should be so let me know what feels good and what you'd like to see and I'll implement from there

**Comment by SlayerOrnstein** [comment]
Alright so assuming DE doesn't look down the endpoint I'm gonna go ahead and get started with manually tracking mods @KerstinMaur

**Comment by KerstinMaur** [comment]
I just had a look and it is awesome! 😍
Thank you so much!
